examples/metalk8s-solution-example/operator/go.sum

```diff
@@ -42,13 +42,15 @@ github.com/Azure/go-autorest/autorest/date v0.2.0 h1:yW+Zlqf26583pE43KhfnhFcdmSW
 github.com/Azure/go-autorest/autorest/date v0.2.0/go.mod h1:vcORJHLJEh643/Ioh9+vPmf1Ij9AEBM5FuBIXLmIy0g=
 github.com/Azure/go-autorest/autorest/mocks v0.1.0/go.mod h1:OTyCOPRA2IgIlWxVYxBee2F5Gr4kF2zd2J5cFRaIDN0=
 github.com/Azure/go-autorest/autorest/mocks v0.2.0/go.mod h1:OTyCOPRA2IgIlWxVYxBee2F5Gr4kF2zd2J5cFRaIDN0=
+github.com/Azure/go-autorest/autorest/mocks v0.3.0 h1:qJumjCaCudz+OcqE9/XtEPfvtOjOmKaui4EOpFI6zZc=
```

Comment:
Not sure why we have all those changes to the `go.sum` of `solution-example` (only additions, no removal).

Reply:
I don't understand how `go.sum` works, TBH... I've removed the code from `pkg/config` and ran `go mod tidy` (I expected this would refresh both `go.mod` and `go.sum` based on the imports within the module), which complains about something entirely unrelated:

```
example-solution-operator imports
        sigs.k8s.io/controller-tools/pkg/crd/generator: module sigs.k8s.io/controller-tools@latest found (v0.4.0), but does not contain package sigs.k8s.io/controller-tools/pkg/crd/generator
```

And that's roughly all I did, so I'm not sure what's expected or what I should do.

Reply:
OK I split the `go mod tidy` into its own commit, to make things clearer.